pkg/dns/aws/dns.go

```diff
@@ -0,0 +1,248 @@
+package aws
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/route53"
+)
+
+var _ dns.Manager = &Manager{}
+
+// Manager is a DNSManager for AWS which is tightly coupled to the DNS zone
```

Comment:
I guess that should be "is a dns.Manager" now.

Reply:
Revised comment